### DISCUSSION TITLE
How to switch tabs inside a panel?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [jesseduffield/lazygit#1861] [Q&A] [answered]
As shown in the picture, I was in the `branch panel` and I want to switch to the `Remotes` or `Tags` tab. 

How could achieve that with the keyboard?

<img width="602" alt="image" src="https://user-images.githubusercontent.com/95092244/161883792-bfcbd4f8-be2a-4167-92ce-ac105354ad88.png">

**Comment by anasrar** [comment, marked as answer]
using `[` to `prev tab` or `]` to `next tab`
you can remap in `config.yml`
```yml
keybinding:
  universal:
    nextTab: ']'
    prevTab: '['
```
https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md

**Reply by LintaoAmons** [reply]
Thanks~~
